dask/array/tests/test_array_core.py

```diff
@@ -918,6 +918,29 @@ def test_elemwise_dtype():
             assert da.maximum(x, y).dtype == da.result_type(x, y)
 
 
+@pytest.mark.parametrize(
+    "x,o",
+    [
+        [
+            [11, 2, 3, 4, 5, 12, 34, 56, 78, 90],
+            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+        ],
+        [
+            [[11, 2, 3, 4, 5], [6, 7, 8, 9, 0], [12, 34, 56, 78, 90]],
+            [[1, 1, 1, 1, 1], [1, 1, 1, 1, 1], [1, 1, 1, 1, 1]],
+        ],
+    ],
+)
+def test_elemwise_return(x, o):
+
+    x = da.array(x, dtype=np.int8)
+    o = da.array(o, dtype=np.int8)
+
+    o_ = da.invert(x, out=o)
+
```

Comment:
an additional `assert o_ is o` line would test that the object returned is the same as the object passed to `out=`